examples/css/timetree_chrome_devtools.css

```diff
@@ -1,60 +0,0 @@
-.chrome-devtools-timetree, .chrome-devtools-timetree .label text {
```

Comment:
Okay, good call. Changing the behavior is out of scope of this PR. But if we want this to ship with the addon at all, gotta bring the file back, right?